db/migrations/008_create_optional_builds.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+    change do
+        create_table(:optional_builds) do
+            DateTime :date, null: false, unique: true
```

Comment:
Industry best practice is to have an integer primary key for any table, so I'd suggest adding one here. The date can still be unique.